app/gui/src/project-view/components/ComponentBrowser/input.ts

```diff
@@ -139,13 +139,18 @@ export function useComponentBrowserInput(
   })
 
   /** Apply given suggested entry to the input. */
-  function applySuggestion(id: SuggestionId): Result {
+  function applySuggestion(id: SuggestionId, label: string): Result {
     const entry = suggestionDb.get(id)
     if (!entry) return Err(`No entry with id ${id}`)
+
+    const matchedLabel = label.replace(/^.+?(?:\((.*)\))?$/g,'$1')
```

Comment:
This logic should be moved; now, we have handy `match.matchedAlias` in [`makeComponent` function](https://github.com/enso-org/enso/blob/develop/app/gui/src/project-view/components/ComponentBrowser/component.ts#L101). So please add a `macroSuffix` field to `Component` type, and compute it in `makeComponent`. Then, this `applySuggestion` could receive the suffix in place of the label.